src/scale.ts

```diff
@@ -415,6 +415,14 @@ export interface ScaleConfig<ES extends ExprRef | SignalRef> {
    * Reverse x-scale by default (useful for right-to-left charts).
    */
   xReverse?: boolean | ES;
+
+  /**
+   * Override the default `scale.zero` for [`continuous`](https://vega.github.io/vega-lite/docs/scale.html#continuous) scale except for non-ranged bar or area charts.
```

Comment:
> [`continuous`](https://vega.github.io/vega-lite/docs/scale.html#continuous) scale

scale**s**

Reply:
> except for non-ranged bar or area charts.

If you agree with the other comment to always do "zero" for size scale. 

except for (1) x/y-scales of non-ranged bar or area charts and (2) size scales.